src/jaxoplanet/light_curves.py

```diff
@@ -66,6 +66,10 @@ def light_curve(
         if texp is None:
             tgrid = t
         else:
+            if (
+                texp.ndim == 0
```

Comment:
Are we sure that `texp` will always have a `.ndim` attribute? What about when a `float` is passed? Maybe we should use `jnp.ndim`?

Reply:
I'm pretty sure `texp` will always have an `.ndim` attribute since it's automatically converted to a `pint.Quantity` object. This behavior is true even if `texp` is a `float`.

But I'm happy to change it!